PrivateCollections/Germany/Neukirchen006.xml

```diff
@@ -0,0 +1,502 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"
+?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0"
+     xml:lang="en"
+     xml:id="Neukirchen006"
+     type="mss">
+  <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>Psalter</title>
+            <editor key="DN"/>
+            <editor role="generalEditor" key="AB"/>
+            <funder>Akademie der Wissenschaften in Hamburg</funder>
+         </titleStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <pubPlace>Hamburg</pubPlace>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                  <p> This file is licensed
+            under the Creative Commons Attribution-ShareAlike 4.0. </p>
+               </licence>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+              <msIdentifier>
+                  <repository ref="INS0915Neukirchen">Private collection of Dr. Carl Heinz
+                    and Jutta Neukirchen</repository>
+                  <idno>Private collection of Dr. Carl Heinz and Jutta Neukirchen 005</idno>
+              </msIdentifier>
+               <msContents>
+                  <summary/>
+                  <msItem xml:id="ms_i1">
+                    <locus from="5r" to="143r"/>
+                     <title type="complete" ref="LIT2701Dawit"/> 
+                     <textLang mainLang="gez"/>
+
+                    <msItem xml:id="ms_i1.1">
+                      <locus from="5r" to="110r"/>
+                      <title type="complete" ref="LIT2000Mazmur"/>
+                      <textLang mainLang="gez"></textLang>
+                      <incipit xml:lang="gez"><hi rend="rubric">
+                        ዘከመ᎓ መዝሙረ፡ ድንግል፡ ትመስል፡ ገነተ።</hi>
```

Comment:
Stupid question, is this a Mazmura dengel?

Reply:
Actually, it is a mention of Mazmura dengel, but it is not that the entire work is distributed between the Psalms. But it can represent a special version of the Psalter. We need to find a couple more witnesses.